src/prettytable.cpp

```diff
@@ -100,7 +116,25 @@ PrettyTable::on_application_closed(WnckScreen* screen, WnckApplication* app, gpo
     static_cast<PrettyTable*>(data)->unregister_application(pid);
 }
 
+void PrettyTable::init_gio_app_cache ()
+{
+    this->gio_apps.clear();
+
+    Glib::ListHandle<Glib::RefPtr<Gio::AppInfo> > apps = Gio::AppInfo::get_all();
```

Comment:
Build warning.
```
prettytable.cpp:123:51: warning: declaration of 'apps' shadows a member of 'PrettyTable' [-Wshadow]

  123 |     Glib::ListHandle<Glib::RefPtr<Gio::AppInfo> > apps = Gio::AppInfo::get_all();

      |                                                   ^~~~

In file included from prettytable.cpp:17:
```
https://travis-ci.org/github/mate-desktop/mate-system-monitor/jobs/711714681#L2259

Reply:
The local variable `apps` has been renamed to `registered_apps`.